### PR TITLE
Created .gitattributes

### PR DESCRIPTION
**Pull request by mdjimy**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+tests/ export-ignore
+tools/ export-ignore
+*.sh eol=lf
```
Comment (dg) [inline .gitattributes:5]: /tools are useful in downloaded ZIP archive

**Comment by mdjimy** [comment]
Sorry, I didn't realized this.

**Comment by dg** [comment]
Can you squash it to single commit?

**Comment by mdjimy** [comment]
Wow, I even didn't know, I can do something like this with a git.
Thanks, I've learned something new again! :)

**Comment by dg** [comment]
Thanks, merged 6003daaf88e31a6f996bbd357684d9a758237cd5

**Pull request closed without merging**